utils/github/__main__.py

```diff
@@ -80,6 +80,8 @@
 
 members = set(github.get_members("ClickHouse", "ClickHouse"))
 def print_responsible(pull_request):
+    if "author" not in pull_request or pull_request["author"] is None:
```

Comment:
In which cases this may happen?

Reply:
I had a case when there was an exception and `pull_request["author"]` was `None`
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/nik-kochetov/dev/ClickHouse/utils/github/__main__.py", line 117, in <module>
    print(f'{CROSS_MARK} {bad["number"]}: {bad["url"]} ({print_responsible(bad)})')
  File "/home/nik-kochetov/dev/ClickHouse/utils/github/__main__.py", line 83, in print_responsible
    if pull_request["author"]["login"] in members:
TypeError: 'NoneType' object is not subscriptable
```
But I have lost exact pr :(